Jenkinsfile

```diff
@@ -63,6 +63,8 @@ pipeline {
                         // Check that Confluence rewrite rules don't duplicate the mistake of adding an extra
                         // '/' after the JENKINS portion of the URL.
                         sh 'if grep JENKINS// dist/profile/templates/confluence/vhost.conf; then echo "Extra / after JENKINS in Confluence URL"; exit 1; fi'
+                        // Check confluence URLs from vhost.conf file
+                        sh 'scripts/verify_confluence_urls'
```

Comment:
since its in a ruby agent, do you want me to rewrite it in ruby so it doesn't have so many magical seds in it?

Reply:
@halkeye I would be delighted if it were rewritten in ruby.  I have no ruby skills and didn't think it was the right approach to write it in Python then call it from a Ruby container. 

I'm sure it would be much more readable and maintainable in ruby than the current ugliness that the Unix shell and utilities provide.  If you're willing, that would be great for me.  I've been able to do some limited maintenance on ruby but have not spent enough time with it to develop any real skills with it.

Reply:
I'm doing ruby full time again, so its a good excuse to do a bit of practice :)